core/capabilities/ccip/ccipevm/gas_helpers_test.go

```diff
@@ -77,7 +77,9 @@ func Test_calculateMessageMaxGas(t *testing.T) {
 				TokenAmounts: getTokenAmounts(t, tt.args.numTokens, tt.args.tokenGasOverhead),
 				ExtraArgs:    tt.args.extraArgs,
 			}
-			ep := EstimateProvider{}
+			// Set the source chain selector to be EVM for now
+			msg.Header.SourceChainSelector = 5009297550715157269
```

Comment:
nit/ maybe use a constant from `chainsel` instead to be clearer, e.g `chainsel.ETHEREUM_TESTNET_SEPOLIA.Selector`?